Makefile

```diff
@@ -31,6 +32,7 @@ O_FILES = $(BUILD_PATH)/_types.o \
 install: $(VIRTUAL_ENV)
 	. $(VIRTUAL_ENV)/bin/activate; \
 		pip install pipenv==2018.11.26; \
+		$(PYTHON) $(SRC_PATH)/unicodeprops.py > $(UNICODEPROPS); \
```

Comment:
Generating UnicodeProps.hpp will either have to be part of setup.py, or done as a step where we check the file into source control (and then we just need a 'make' target to update it periodically).

Reply:
Thanks, will do.